client/transactions/list/index.tsx

```diff
@@ -163,10 +166,41 @@ const getColumns = (
 			required: true,
 			isLeftAligned: true,
 		},
+		{
+			key: 'customer_currency',
+			label: __( 'Customer Currency', 'woocommerce-payments' ),
+			screenReaderLabel: __(
+				'Customer Currency',
+				'woocommerce-payments'
+			),
+			isSortable: true,
+			visible: false,
+		},
+		{
+			key: 'customer_amount',
+			label: __( 'Customer Amount', 'woocommerce-payments' ),
+			screenReaderLabel: __(
+				'Amount in Customer Currency',
+				'woocommerce-payments'
+			),
+			isNumeric: true,
+			isSortable: true,
+			visible: false,
+		},
+		{
+			key: 'deposit_currency',
+			label: __( 'Deposit Currency', 'woocommerce-payments' ),
+			screenReaderLabel: __( 'Deposit Currency', 'woocommerce-payments' ),
+			isSortable: true,
+			visible: false,
+		},
 		{
 			key: 'amount',
 			label: __( 'Amount', 'woocommerce-payments' ),
```

Comment:
@mordeth @souravdebnath1986 I wonder if we should rename this to "Deposit Amount" like we did in the server-exported CSV.

Reply:
@anu-rock I intentionally refrained from modifying the `Amount` label to avoid the column expanding too much, which could lead to other columns being displaced and result in an unattractive layout.

How about incorporating a small helper icon and describing the label in a tooltip? What do you think about that?

Reply:
@mordeth As we discussed during our last sync team chat, I am aligned with your suggestion about using a tooltip. We can check with Design before making a final call.